packages/lesswrong/components/posts/EALargePostsItem.tsx

```diff
@@ -0,0 +1,204 @@
+  noImagePlaceholder?: boolean,
+  className?: string,
+  classes: ClassesType,
+}) => {
+  const authorExpandContainer = useRef(null);
+
+  const postLink = postGetPageUrl(post);
+  const {onClick} = useClickableCell({href: postLink});
+
+  const {eventHandlers} = useHover({
+    pageElementContext: "postListItem",
+    documentId: post._id,
+    documentSlug: post.slug,
+  });
+
+  const timeFromNow = moment(new Date(post.postedAt)).fromNow();
+  const ago = timeFromNow !== "now"
+    ? <span className={classes.xsHide}>&nbsp;ago</span>
+    : null;
+
+  let imageUrl = post.socialPreviewData.imageUrl;
+  if (!imageUrl && !noImagePlaceholder) {
+    imageUrl = siteImageSetting.get();
+  }
+
+  const {TruncatedAuthorsList, ForumIcon, PostsItemTooltipWrapper} = Components;
+  return (
+    <AnalyticsContext documentSlug={post.slug}>
+      <div
+        {...eventHandlers}
+        onClick={onClick}
+        className={classNames(classes.postListItem, className)}
+      >
+        <div className={classes.postListItemTextSection}>
+          <div className={classes.postListItemTitle}>
+            <PostsItemTooltipWrapper post={post} placement="bottom" As="span">
+              <InteractionWrapper>
+                <Link to={postLink}>{post.title}</Link>
+              </InteractionWrapper>
+            </PostsItemTooltipWrapper>
+          </div>
+          {/** TODO
+            * The recent discussions redesign adds an `EAPostMeta` component
+            * which should probably be used here:
+            * https://github.com/ForumMagnum/ForumMagnum/pull/7858/files#diff-56fb35d84b446595acd8a1318777ab5c5d4f1d47434941f5baf9e2949c24431d
+            */}
+          <div className={classes.postListItemMeta}>
+            <div ref={authorExpandContainer}>
+              <InteractionWrapper>
+                <TruncatedAuthorsList
+                  post={post}
+                  expandContainer={authorExpandContainer}
+                  className={classes.authors}
+                />
+              </InteractionWrapper>
+            </div>
+            &nbsp;·&nbsp;
+            {timeFromNow}
+            {ago}
+            &nbsp;·&nbsp;
+            {post.readTimeMinutes}m read
+            <div>
+              {!isNarrow && (
+                <span className={classNames(classes.commentCount, classes.xsHide)}>
+                  &nbsp;·&nbsp;
+                  <InteractionWrapper>
+                    <Link to={`${postLink}#comments`} className={classes.commentCount}>
+                      <ForumIcon icon="Comment" />
+                      {post.commentCount}
+                    </Link>
+                  </InteractionWrapper>
+                </span>
+              )}
+            </div>
+          </div>
+          <div className={classes.postListItemPreview}>
+            {post.contents?.plaintextDescription}
+          </div>
+        </div>
+        {imageUrl && <img className={classes.postListItemImage} src={imageUrl} />}
```

Comment:
This image isn't loading for me:
<img width="828" alt="Screenshot 2023-09-25 at 10 31 30" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/06c56792-d5c2-4637-b62e-46b5fbc2ead8">
We used to have this bug in posts too, it's caused by google hosted images (with urls https://lh3.googleusercontent.com/...) getting rate limited. It is fixed in posts now because we rehost all the images in cloudinary with an async callback, but I expect the social preview image is set before that so it can still happen here. Could you change the callback to also update the social preview image? It's in `packages/lesswrong/server/editor/make_editable_callbacks.ts`

Reply:
Thanks - had no idea about this!

I've tested it, but would appreciate a quick sanity check on commit https://github.com/ForumMagnum/ForumMagnum/pull/7883/commits/8f74ebbf332af384706fc39d1016e0ec1d0bcb17